app/presenters/organisations/student_presenter.rb

```diff
@@ -130,6 +133,27 @@ def filters_in_url
         .compact
     end
 
+    def milestone_completion_stats
+      stats = {}
+
+      status = milestone_completion_status
+
+      stats[:completed_milestones_count] = status.values.count do |target|
+        target[:completed]
+      end
+
+      stats[:percentage] = (
+        (stats[:completed_milestones_count] / total_milestone_targets.to_f) *
+          100
+      ).round
+
+      stats
+    end
+
+    def total_milestone_targets
```

Comment:
Maybe this method is not required. This could just be an internal variable inside the method you are doing the computation if the idea is readability